doc/classes/AudioEffectFilter.xml

```diff
@@ -14,6 +14,7 @@
 			Threshold frequency for the filter, in Hz.
 		</member>
 		<member name="db" type="int" setter="set_db" getter="get_db" enum="AudioEffectFilter.FilterDB" default="0">
+			Steepens of the cutoff curve in dB per octave, also known as the order of the filter. Higher orders have a more aggressive cutoff.
```

Comment:
```suggestion
			Steepness of the cutoff curve in dB per octave, also known as the order of the filter. Higher orders have a more aggressive cutoff.
```
My bad corrected the typo the wrong way

Reply:
Oh right. this was the kind of thing I tend to mess up so I just did not double-check. Good catch.